docs/guide/development/pipeline-step.md

```
@@ -40,50 +41,54 @@ public class ProcessPaymentService implements ReactiveService<PaymentRecord, Pay
+   - `ReactiveStreamingClientService<I, O>`
+   - `ReactiveBidirectionalStreamingService<I, O>`
+4. Keep YAML cardinality aligned with the implemented reactive interface.
 
 Parallelism is configured at the pipeline level (`pipeline.parallelism` and `pipeline.max-concurrency`).
 The `ordering` and `threadSafety` values on `@PipelineStep` are propagated to the generated client step,
 which the runtime uses to decide parallelism under `AUTO`.
 
-Transport selection (gRPC vs REST) is configured globally in `pipeline.yaml`, not on the annotation.
-
-The framework automatically generates and registers the adapter beans at build time.
+Transport selection (gRPC vs REST) is configured in `pipeline.yaml`, not on the annotation.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)